ClosedXML/Excel/XLWorkbook_Save.cs

```diff
@@ -2076,6 +2076,37 @@ private static void GenerateTableDefinitionPartContent(TableDefinitionPart table
                 pivotTableCacheDefinitionPart.PivotCacheDefinition = pivotCacheDefinition;
             }
 
+            #region CreatedVersion
+
+            byte createdVersion = 5;
+
+            if (pivotCacheDefinition.CreatedVersion?.HasValue ?? false)
+                pivotCacheDefinition.CreatedVersion = Math.Max(createdVersion, pivotCacheDefinition.CreatedVersion.Value);
+            else
+                pivotCacheDefinition.CreatedVersion = createdVersion;
```

Comment:
Maybe something like this?

```
byte createdVersionClosedXml = 5;
var createdVersionFromFile = pivotCacheDefinition.CreatedVersion.GetValueOrDefault();
pivotCacheDefinition.CreatedVersion = Math.Max(createdVersionFromFile, createdVersionClosedXml);
```

Or even extract `createdVersion`, `refreshedVersion` and `minRefreshableVersion` to some class with constants?

Reply:
The existing pivot table reference files implicitly test this, but yes, I'll add an explicit test.

Reply:
`pivotCacheDefinition.CreatedVersion` is of type `ByteValue`, not `byte?` and hence doesn't have a `GetValueOrDefault()` method.